iroh-willow/src/proto/grouping.rs

```diff
@@ -93,6 +95,7 @@ impl AreaExt for Area {
 /// A single point in the 3D range space.
 ///
 /// I.e. an entry.
+// TODO: Upstream to willow-rs.
```

Comment:
Yes these are really good TODOs. Also need this on `PathExt` and `EntryExt` in the `data_model.rs` file.